cmd/di.go

```diff
@@ -91,6 +92,7 @@ type Dependencies struct {
 	StatisticsTracker  *statistics.SessionStatisticsTracker
 	StatisticsReporter *statistics.SessionStatisticsReporter
 	SessionStorage     *consumer_session.Storage
+	EventsSender       *events.Sender
```

Comment:
Is it really an EventsSender? To me it seems more appropriate to name it Metrics.Sender or something like that. Maybe rename the whole events package to metrics?

Reply:
I was thinking about that, but we already have `metrics` package (`market.metrics`), so naming 2 things with same package names seemed confusing and importing both packages will require named imports.

Reply:
Renamed to `metrics`